akirachix/Inganji-Informational
# Added Footer

This pull request introduces a new footer component for the Fanikisha Informational website. 

The motivation for this change is to enhance the user experience by providing quick access to important links and information, such as contact details and social media links. A consistent footer will help improve site navigation and user engagement.

* What is this?
This PR adds a footer component to the main layout of the fanikisha website.
* Why this?
A footer is essential for providing users with easy access to important links and information, which improves overall site usability.
* How are you doing this? 
We are implementing this by creating a new Footer component in Next.js, which includes links to the website's navigation bar and contact information.

- List of updates for features, enhancements, or bugfixes
Created a responsive Footer component
Added contact and location information
Ensured the footer is styled to match the website’s branding
Updated the main layout to include the Footer component


- [ yes] New feature (non-breaking change which adds functionality)

## How can it be tested?
Check out the branch for this PR.
Run the application using npm run dev.
Navigate to various pages on the site.
Click the links in the footer to ensure they lead to the correct pages.

# Where is it documented?

https://ivylovelace.atlassian.net/browse/IFYP-106
https://www.figma.com/design/4djg9iIwFikQoWeQo8iGde/Fanikisha-UI?node-id=1-754&t=ypr7RTlkhFXrcmyV-1
